__tests__/admin.test.js

```diff
@@ -0,0 +1,123 @@
+
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '..';
+import seededUsers from './mockData/seededUsers';
+import mockUsers from './mockData/mockUsers';
+
+chai.use(chaiHttp);
+
+const url = '/api/v1/admin/users';
+const { expect } = chai;
+let verifiedUserToken;
+const userId = '356304da-50bc-4488-9c85-88874a9efb16';
+
+describe('Admin Routes', () => {
+  before((done) => {
+    chai.request(app)
+      .post('/api/v1/auth/signup')
+      .send(mockUsers[15])
+      .end(() => { });
+    chai.request(app)
+      .post('/api/v1/auth/signin')
+      .send(seededUsers[3])
+      .end((err, res) => {
+        const { token } = res.body.user;
+        verifiedUserToken = token;
+        done();
+      });
+  });
+  it('should successfully get all users', (done) => {
+    chai
+      .request(app)
+      .get(`${url}`)
+      .set('Authorization', `Bearer ${verifiedUserToken}`)
+      .end((err, res) => {
+        expect(res.status).to.eql(200);
+        expect(res.body.users[0]).to.have.property('id');
+        expect(res.body.users[0]).to.have.property('firstName');
+        expect(res.body.users[0]).to.have.property('lastName');
```

Comment:
No specific assertion here?

Reply:
> No specific assertion here?

No. that test suite is to get all the users